packs/data/fall-of-plaguestone.db/vilree.json

```diff
@@ -657,12 +657,12 @@
                 }
             },
             "img": "systems/pf2e/icons/equipment/alchemical-items/alchemical-bombs/acid-flask.webp",
-            "name": "Acid Flask (Moderate) [Infused]",
+            "name": "Acid Flask (Moderate) [Infusion]",
```

Comment:
Same as the other PR, these should say `Infused`. But @SpartanCPA may want us to use parenthesis instead.